### ISSUE TITLE
add mqtt username and password fields to sites.json

### ISSUE THREAD

**Issue by abzman** [i3detroit/Automatic-IoT-module-programming#47]
I can't make the homeassistant OS install of mosquitto cooperate with the DEVS_USER username so I have had to manually change all of them in the web interface.  I know that's the issue because in previous versions of tasmota or our script the username was '0' and that worked.

**Comment by mkfink** [comment]
I think you misread. It's "DVES_USER". DVES, not DEVS. Could that be the issue?

**Comment by abzman** [comment]
No, I had it set up so any username and password could connect and some did.  Just not the ones with that username.   Even if I carved out a special inatance for that username they still didn't connect.  The older flashed devices that did not use that name connected fine though.  After renaming them through the web interface they also connected fine as long as the username was not that.
